vendor/rivercity-innovations/lynx-codec.yaml

```diff
@@ -0,0 +1,74 @@
+# Uplink decoder decodes binary data uplink into a JSON object (optional)
+# For documentation on writing encoders and decoders, see: https://thethingsstack.io/integrations/payload-formatters/javascript/
+uplinkDecoder:
+  fileName: lynx.js
+  # Examples (optional)
+  examples:
+    - description: Regular Uplink
+      input:
+        fPort: 2
+        bytes: [0x03, 0x1B, 0xA1, 0x77, 0xF9, 0xA4, 0xFE, 0xF1, 0x94, 0xDF, 0x26]
+      output:
```

Comment:
Please review the input is not matching the output:

`"data":{"batV":4,"direction":314.8235294117647,"fixFailed":false,"inTrip":true,"latitudeDeg":52.142455,"longitudeDeg":-106.627343,"temperature":19,"type":"position"}}`
 does not match 
`{"data":{"latitudeDeg":52.142455,"longitudeDeg": -106.627343,"inTrip":true,"fixFailed":false,"batV":4,"direction":314.8,"temperature":19}}`